src/bundle/Resources/public/js/CKEditor/link/ui/link-form-view.js

```diff
@@ -16,16 +16,56 @@ class IbexaLinkFormView extends View {
         this.selectContentButtonView = this.createButton('Select content', null, 'ibexa-btn--select-content');
         this.urlInputView = this.createTextInput('Link to');
         this.titleView = this.createTextInput('Title');
-        this.targetSwitcherView = this.createBoolean('Open in');
+        this.targetSwitcherView = this.createBoolean('Open in tab');
 
         this.children = this.createFormChildren();
 
         this.setTemplate({
-            tag: 'form',
+            tag: 'div',
             attributes: {
-                tabindex: '-1',
+                class: 'ibexa-ckeditor-balloon-form',
             },
-            children: this.children,
+            children: [
+                {
+                    tag: 'div',
+                    attributes: {
+                        class: 'ibexa-ckeditor-balloon-form__header',
+                    },
+                    children: ['Link'],
+                },
+                {
+                    tag: 'form',
+                    attributes: {
+                        tabindex: '-1',
+                    },
+                    children: [
+                        {
+                            tag: 'div',
+                            attributes: {
+                                class: 'ibexa-ckeditor-balloon-form__fields',
+                            },
+                            children: [
+                                this.children.first,
+                                {
+                                    tag: 'div',
+                                    attributes: {
+                                        class: 'ibexa-ckeditor-balloon-form__separator',
+                                    },
+                                    children: ['Or'],
+                                },
+                                ...this.children.filter((child, index) => index !== 0),
```

Comment:
```suggestion
                                ...this.children.slice(1),
```
Wouldn't be better and faster?

Reply:
Would be if this would be an array ;)
This is ckeditor collection.